transport/http2_client.go

```diff
@@ -135,9 +135,14 @@ func newHTTP2Client(ctx context.Context, addr string, opts ConnectOptions) (_ Cl
 		conn, authInfo, connErr = creds.ClientHandshake(ctx, addr, conn)
 	}
 	if connErr != nil {
+		temp := false
+		switch connErr {
+		case io.EOF, context.DeadlineExceeded:
```

Comment:
Done.

On Mon, Aug 22, 2016 at 2:25 PM, Qi Zhao <notifications@github.com> wrote:

> In transport/http2_client.go
> <https://github.com/grpc/grpc-go/pull/846#discussion_r75731294>:
>
> > @@ -135,9 +135,14 @@ func newHTTP2Client(ctx context.Context, addr string, opts ConnectOptions) (_ Cl
> >  		conn, authInfo, connErr = creds.ClientHandshake(ctx, addr, conn)
> >  	}
> >  	if connErr != nil {
> > +		temp := false
> > +		switch connErr {
> > +		case io.EOF, context.DeadlineExceeded:
>
> Please add some comments.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc-go/pull/846/files/dba1af305f643090367ac595c5eb41819b092c98#r75731294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPOSZvO419HS6_WZ6BcFvWJrEUKVWks5qiemagaJpZM4Jo-ij>
> .
>